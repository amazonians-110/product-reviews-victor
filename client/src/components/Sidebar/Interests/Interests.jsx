import React from 'react';

class Interests extends React.Component {
  constructor(props) {
    super(props);
  }

  render() {
    return(
      <div>
        <b>By consumer groups & interests</b>
        <div>Headphones       ***** 4.2</div>
        <div>Customers in New York    ***** 4.7</div>
        <div>Is this feature helpful? 
          <button>Yes</button>
          <button>No</button>
        </div>        
      </div>
    )
  }
}

export default Interests;