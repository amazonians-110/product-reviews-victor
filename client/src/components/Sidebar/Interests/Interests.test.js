import React from 'react';
import { shallow } from 'enzyme';

import Interests from './Interests';

describe('Interests', () => {
  it('should render correctly', () => {
    
    const component = shallow(<Interests/>);

    expect(component).toMatchSnapshot();
  });
});
