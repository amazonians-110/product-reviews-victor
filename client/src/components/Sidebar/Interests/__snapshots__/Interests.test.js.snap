// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Interests should render correctly 1`] = `
<div>
  <b>
    By consumer groups & interests
  </b>
  <div>
    Headphones       ***** 4.2
  </div>
  <div>
    Customers in New York    ***** 4.7
  </div>
  <div>
    Is this feature helpful?
    <button>
      Yes
    </button>
    <button>
      No
    </button>
  </div>
</div>
`;
